src/frontend/src/handler/create_source.rs

```diff
@@ -998,6 +998,9 @@ static CONNECTORS_COMPATIBLE_FORMATS: LazyLock<HashMap<String, HashMap<Format, V
                 NATS_CONNECTOR => hashmap!(
                     Format::Plain => vec![Encode::Json, Encode::Protobuf],
                 ),
+                MQTT_CONNECTOR => hashmap!(
+                    Format::Plain => vec![Encode::Json, Encode::Bytes],
```

Comment:
Is `format plain encode bytes` on `source` tested?

Reply:
I did, and this is the output
```

dev=> CREATE TABLE mqtt_source_table                                                                                                                                                                                                                       (                                                                                                                                                                                                                                                            value bytea                                                                                                                                                                                                                                              )                                                                                                                                                                                                                                                          WITH (                                                                                                                                                                                                                                                         connector='mqtt',                                                                                                                                                                                                                                          url='tcp://localhost',                                                                                                                                                                                                                                     topic= 'test',                                                                                                                                                                                                                                             qos = 'at_least_once',                                                                                                                                                                                                                                 ) FORMAT PLAIN ENCODE BYTES;
CREATE_TABLE

dev=> SELECT * FROM mqtt_source_table;
                         value
--------------------------------------------------------
 \x7b226964223a352c226e616d65223a224172616d696e7461227d
 \x7b226964223a342c226e616d65223a224a65727279227d
 \x7b226964223a322c226e616d65223a22426f62227d
 \x7b226964223a362c226e616d65223a22436c6f766572227d
 \x7b226964223a332c226e616d65223a22546f6d227d
 \x7b226964223a312c226e616d65223a22416c696365227d
 \x7b226964223a382c226e616d65223a2257617665726c79227d
 \x7b226964223a372c226e616d65223a22506f736579227d
(8 rows)

dev=> select encode(value, 'escape') from mqtt_source_table limit 1;
           encode
----------------------------
 {"id":5,"name":"Araminta"}
(1 row)

dev=>
```